ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -53,6 +53,10 @@ def create_timer(self, fire_at: Union[datetime, timedelta]) -> task.Task:
 
     def call_activity(self, activity: Callable[[WorkflowActivityContext, TInput], TOutput], *,
                       input: TInput = None) -> task.Task[TOutput]:
+        if hasattr(activity, '_alternate_name'):
+            return self.__obj.call_activity(activity=activity.__dict__['_alternate_name'],
+                                            input=input)
+        # this return should ideally never execute
```

Comment:
Why is this "ideal"? Isn't this the case where the user chose not to use the decorator, which we plan to allow?

Reply:
No in both scenarios, the attribute `_alternate_name` will be set for the function being called. So ideally the if clause will always execute.